lib/mix/tasks/word2vec/compile.ex

```diff
@@ -44,8 +44,8 @@ defmodule Mix.Tasks.Word2vec.Compile do
 
       options:
         --partitions:  number of partitions (files) to create, default: 1
-        --size_hint:   index hint for the total number of words
-        --vector_size: number of vectors/word, default: 300
+        --size-hint:   index hint for the total number of words
```

Comment:
thanks for fixing this, I have been burned by it

Reply:
Yeah, me too...took me awhile to find that little gotcha in the docs for `OptionParser`.